1-js/01-getting-started/2-code-editors/article.md

```diff
@@ -51,12 +51,12 @@ I'm using:
 - [WebStorm](http://www.jetbrains.com/webstorm/) for JS, and if there is one more language in the project, then I switch to other Jetbrains editors like [PHPStorm](http://www.jetbrains.com/phpstorm/) (PHP), [IDEA](http://www.jetbrains.com/idea/) (Java), [RubyMine](http://www.jetbrains.com/ruby/) (Ruby). There are editors for other languages too, but I haven't used them.
 - As a lightweight editor -- [Sublime Text](http://www.sublimetext.com) or [Atom](https://atom.io/).
 
-If you don't know what to choose -- you can consider these ones.
+If you don't know what to choose, you can consider these ones.
 
 ## Let's not argue
 
-The editors in the lists above are those that me or my friends -- good developers have been using for a long time and are happy with.
+The editors in the lists above are those that either my friends or I -- good developers -- have been using for a long time and are happy with.
```

Comment:
What about: "...that I or my friends who I consider good developers have been using for a long time" ?